src/main/java/com/spotinst/sdkjava/example/elastigroup/aws/ElastigroupUsageExample.java

```diff
@@ -1231,4 +1251,66 @@ private static ElastigroupGetInstanceStatusResponse getInstanceStatus(SpotinstEl
         return response;
     }
 
+    private static String getBeanstalkConfig(SpotinstElastigroupClient client, String environmentId, String region) {
+
+        System.out.println(String.format("Get Beanstalk Configuration. environmentId: %s region: %s", environmentId, region));
+        Elastigroup elastigroup = client.getBeanstalkConfig(environmentId, region);
+        // Build elastigroup creation request
+        ElastigroupCreationRequest.Builder elastigroupCreationRequestBuilder = ElastigroupCreationRequest.Builder.get();
+        ElastigroupCreationRequest creationRequest =
+                elastigroupCreationRequestBuilder.setElastigroup(elastigroup).build();
+
+        // Convert elastigroup to API object json
+        System.out.println(creationRequest.toJson());
+
+        // Create elastigroup
+        Elastigroup createdElastigroup = client.createElastigroup(creationRequest);
+        System.out.println("Elastigroup succesfully created: " + createdElastigroup.getId());
+
+        // Get elastigroup Id
+        return createdElastigroup.getId();
+    }
+
+    private static Boolean startBeanstalkMaintenance(SpotinstElastigroupClient client, String groupId) {
+
+        System.out.println(String.format("Start beanstalk maintenance. groupId: %s", groupId));
+        Boolean response = client.startBeanstalkMaintenance(groupId);
+
+        System.out.println(String.format("Start beanstalk maintenance response: %s", response));
+
+        return response;
+    }
+
+    private static String getBeanstalkMaintenanceStatus(SpotinstElastigroupClient client, String groupId) {
+
+        System.out.println(String.format("Start beanstalk maintenance. groupId: %s", groupId));
+        ElastigroupGetBeanstalkMaintenanceStatusResponse response = client.getBeanstalkMaintenanceStatus(groupId);
+
+        System.out.println(String.format("Beanstalk maintenance status: %s", response.getStatus()));
+
+        return response.getStatus();
+    }
+
+    private static Boolean finishBeanstalkMaintenance(SpotinstElastigroupClient client, String groupId, String currentStatus) {
+
+        System.out.println(String.format("Finish beanstalk maintenance. groupId: %s", groupId));
```

Comment:
change the print statement to "Finish beanstalk maintenance of the group: %s with current status: %s"